README.md

```diff
@@ -79,198 +83,58 @@ requires the [CUDAnative.jl](https://github.com/JuliaGPU/CUDAnative.jl) package.
 
 * Tier 4: Julia is known not to build.
 
-It is possible that Julia will build and work on other platforms too, and we're always
-looking to improve our platform coverage.
-If you're using Julia on a platform not listed here, let us know!
+It is possible that Julia will build and work on other platforms too,
+and we're always looking to improve our platform coverage.  If you're
+using Julia on a platform not listed here, let us know!
 
 ## Source Download and Compilation
```

Comment:
Maybe rename to `Source Compilation`;  would be simpler and as equally descriptive. 

```suggestion
## Source Compilation
```